### ISSUE TITLE
instance_geometry_set_shader_uniform fails when building

### ISSUE THREAD

**Issue by Eirikalv1** [Zylann/godot_voxel#438]
Just got the latest build of godot 4. I get this error when building voxel tools with godot 4:

```
modules/voxel/util/godot/direct_mesh_instance.cpp: In member function 'void zylann::DirectMeshInstance::set_shader_instance_uniform(StringName, Variant)':
modules/voxel/util/godot/direct_mesh_instance.cpp:100:12: error: 'class RenderingServer' has no member named 'instance_geometry_set_shader_uniform'; did you mean 'instance_geometry_set_shader_parameter'?
  100 |         vs.instance_geometry_set_shader_uniform(_mesh_instance, key, value);
      |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |            instance_geometry_set_shader_parameter
scons: *** [modules/voxel/util/godot/direct_mesh_instance.linuxbsd.tools.x86_64.o] Error 1
scons: building terminated because of errors.
[Time elapsed: 00:00:26.372]
```

**Comment by Zylann** [comment]
Godot devs are renaming things every day so the build (and all my ported plugins) keep breaking. `shader_param` occurrences were once renamed to `shader_uniform` but yesterday they merged https://github.com/godotengine/godot-proposals/issues/5123 which reverses the change, but not really since now it's called `shader_parameter` except not in all places, some still use `shader_uniform` for some reason...

**Comment by Zylann** [comment]
Fixed in e14b6fdab00c6f06066f58e164b05371c38bcc3e
